### ISSUE TITLE
Setting Model to eval() mode in test.py

### ISSUE THREAD

**Issue by rllyryan** [nikhilbarhate99/PPO-PyTorch#57]
Dear @nikhilbarhate99,

Can I ask if we need to set the models to evaluation mode (nn.Module.eval()) in the test script to see if the weights are suitable?

Thank you!
